assets/js/pjaxinit.js

```diff
@@ -20,6 +21,16 @@
       },
     },
     cacheBust: false,
+    scrollRestoration: false,
+    scrollTo: disableScroll === true ? false : 0,
+    analytics: function () {
```

Comment:
No need for this. GA4 automatically tracks Pjax visits, right?

Reply:
Removed unused GA4 tracking as it's not needed. GA4 automatically tracks Pjax visits.